.github/workflows/nodejs.yml

```diff
@@ -34,12 +32,12 @@ jobs:
       - name: Use Node.js 14
         uses: actions/setup-node@v2
         with:
-          node-version: 14.18
+          node-version: 14.18.X
       - name: npm ci no scripts
         uses: nick-invision/retry@v2
         with:
           timeout_minutes: 20
```

Comment:
should this be restored to 10 too? (based on what changed from https://github.com/digidem/mapeo-desktop/commit/c989c41cbe0471ed83952e83bf1b3ba0572f1cd4#diff-e98936aa52a6dd7416e4296e9628456227d834f7245967383fd9ff80fd985dad)

Reply:
addressed: https://github.com/digidem/mapeo-desktop/pull/716/commits/53fb8081749a88d7ea2e2b5390eab68fd44adf9f